dependabot/update_lag_status.py

```diff
@@ -0,0 +1,335 @@
+import json
+import sys
+import os
+import time
+
+import github
+from github import Github, InputGitAuthor, GithubException
+from datetime import datetime
+import urllib.request
+
+import constants
+
+ballerina_bot_username = os.environ[constants.ENV_BALLERINA_BOT_USERNAME]
+ballerina_bot_token = os.environ[constants.ENV_BALLERINA_BOT_TOKEN]
+ballerina_bot_email = os.environ[constants.ENV_BALLERINA_BOT_EMAIL]
+ballerina_reviewer_bot_token = os.environ[constants.ENV_BALLERINA_REVIEWER_BOT_TOKEN]
+
+ENCODING = "utf-8"
+ORGANIZATION = "ballerina-platform"
+EXTENSIONS_LIST_FILE = "dependabot/resources/extensions.json"
+BALLERINA_LANG_VERSION_FILE = "dependabot/resources/latest_ballerina_lang_version.json"
+PROPERTIES_FILE = "gradle.properties"
+README_FILE = "README.md"
+LANG_VERSION_KEY = "ballerinaLangVersion"
+BALLERINA_DISTRIBUTION = "ballerina-distribution "
+github = Github(ballerina_bot_token)
+
+all_modules = []
```

Comment:
I see this is used only within one method, any reason to make a global variable?

Reply:
Moved the variable inside the method
4d644cf388a3cd5d458de4ba388ea286a4ab476f